docs/TutorEnvironmentManager.rst

```diff
@@ -0,0 +1,126 @@
+Tutor Environment Manager
+#########################
+
+Tutor environment manager allows you to create one project and set one version for it.
+
+You will be able to continue using the `tutor version manager`_ commands in your project:
+
+- tvm list
+- tvm plugins list
+
+
+Creating a project
+------------------
+
+To create a new project you should call ``tvm project init``, this configures a new tvm project in the current path and set your-project as ``TUTOR_ROOT`` and ``TUTOR_PLUGINS_ROOT``.  
+
+You can create different projects with the same tutor version, you have 2 options:
+
+**Note:** You could use ``tvm project init project-name``, now TVM will create a new folder with that name and in this folder host the project.
+
+If ``project-name`` isn't specified TVM will create a project in the current folder and configure it with that name.
+
+- Use the global active version.
+
+.. code-block:: bash  
+    
+    # Using the global active version
+    
+    mkdir your-project && cd your-project
+    tvm project init
```

Comment:
```suggestion
    tvm project init project-name
```
With that command the folder is already created.

Everything else from the environment manager seems fine to me.